pkgs/by-name/ca/catppuccin-sddm/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, libsForQt5
+, flavors ? [ "latte" "frappe" "macchiato" "mocha" ]
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "catppuccin-sddm";
+  version = "unstable-2023-12-05";
```

Comment:
Due to new guidelines, I believe this should be `0-unstable-2023-12-05`

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming